core/src/main/java/hudson/scheduler/CronTabList.java

```diff
@@ -71,6 +75,16 @@ public String checkSanity() {
         return null;
     }
 
+    public static String getValidTimezone(String timezone) {
```

Comment:
<code>@CheckForNull</code> is a must in such methods. Javadoc would be also useful